src/lib/db/index.ts

```diff
@@ -57,6 +67,16 @@ export interface DemandWithAttachmentRow {
   originalTokenId: number
 }
 
+const demandCommentsColumns = ['id', 'owner', 'state', 'attachment AS attachmentId', 'created_at AS createdAt']
+
+export interface DemandCommentRow {
```

Comment:
missing `updatedAt`?

Reply:
it is but we never use it. My preference is to leave it off now until we need it